src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibLeaseCoordinator.java

```diff
@@ -292,6 +292,17 @@ public Checkpoint getCheckpointObject(String shardId) throws KinesisClientLibExc
         }
     }
 
+    public Set<String> getChildShardIds(String shardId) {
+        String errorMessage = "Unable to fetch childShardIds for shardId " + shardId;
+        try {
+            KinesisClientLease lease = leaseManager.getLease(shardId);
+            return lease.getChildShardIds();
```

Comment:
Are we sure lease can't be null?

Reply:
Good catch, will check null and throw exception